java/fury-core/src/main/java/org/apache/fury/serializer/collection/CollectionSerializers.java

```diff
@@ -255,6 +256,42 @@ public List<?> xread(MemoryBuffer buffer) {
     }
   }
 
+  public static class CopyOnWriteArrayListSerializer
+      extends CollectionSerializer<CopyOnWriteArrayList> {
+
+    private static final long ARRAY_OFFSET;
+
+    static {
+      try {
+        final Field field = CopyOnWriteArrayList.class.getDeclaredField("array");
```

Comment:
Could we implement this serializer like ImmutableListSerializer. The array field here won't be referenced by other objects. So we can use collection serialization protocol